concepts/references/introduction.md

```diff
@@ -0,0 +1,64 @@
+# Introduction
+
+## Copies and References
+
+With our current knowledge, it is a bit cumbersome to update a variable with the help of a function.
+Every argument is handled like a copy and does not change the original.
+Depending on the size of your type, this can have serious consequences for the performance of your code.
+
+```cpp
+int adjust_salary(int base, int kids) {
+    return current + kids * 500;
+}
+int now_a_mother_of_twins{2500};
+adjust_salary(now_a_mother_of_twins, 2);
+
+// now_a_mother_of_twins is still 2500
+now_a_mother_of_twins = adjust_salary(now_a_mother_of_twins, 2);
+// now_a_mother_of_twins is now 3500
+```
+
+`References` can be seen like aliases, changes to them have an effect on the original variable.
+`References` use the **ampersand** in the type declaration: `&`.
+
+```cpp
+int balance{1000};
+int& budget{balance};
+int pro_computer_wheels{699};
+budget -= pro_computer_wheels;
+// budget is now 301;
+// balance is also 301;
+```
+Reseating - changing the binding of a `reference` is not possible.
+You cannot have an unitialized `reference`, you always need a binding.
```

Comment:
IMHO, `initialize` make things more confusing - maybe
 "it always needs to be connected to a variable"?  
Or perhaps
 "it always needs to reference (connect to) a variable"?

Reply:
"initialization" is a technical term in C++ (and is a surprisingly complex topic, one of the top experts wrote a whole [book](https://www.cppstories.com/2022/cpp-init-book/) about it).  
it was mentioned briefly in the "Basics" concept.

"connect" on the other hand is not a well-defined term in C++ (I think)

Reply:
I think for beginners, "initialize" often means "starter value".  But once initialized, `references` can't be changed - or at least that is what I am grokking from the text.  Hence my reticence in using it.  Binding feels less confusing to me, since "bind" in English means "to tie together".

Reply:
Yes, "starter value" sounds about right (colloquially speaking). It's just that you have to provide that starter value (you can't leave references uninitialized), and you cannot re-assign the reference so that it refers to a different object.  
The term "binding" is not wrong, we just haven't introduced it yet.

Reply:
I always had it as binding in my head, but some research suggests, that it is not as official as I thought it to be.
Students will be used to initializations by now. I would not go with "starter value" because that sounds like it is a temporary thing that might change.